posthog/models/session_replay_event/sql.py

```diff
@@ -0,0 +1,118 @@
+from django.conf import settings
```

Comment:
This is 100% personal preference, but I prefer to 1. dial it down on the layering of generating the SQL. I find all the lambdas and .formats etc very convoluted when essentially we are looking for a pretty static SQL string, and 2. add in some comments about e.g. why the table schema is the way it is. Only if it's helpful though of course.

Something like https://github.com/PostHog/posthog/blob/master/posthog/models/person_overrides/sql.py would be my preference.

Enjoy that with your pinch of salt.